cppcache/include/geode/PdxSerializable.hpp

```diff
@@ -34,6 +34,9 @@ namespace client {
 class PdxReader;
 class PdxWriter;
 
+/**
+ * An interface for objects whose state can be written/read as PDX.
```

Comment:
Looks like this was copied form DataSerializable before it had been changed.

"An interface for objects whose contents can be serialized as primitive types."